setup.py

```diff
@@ -1,13 +1,9 @@
 # type: ignore
-import glob
-import importlib.util
-
 from setuptools import setup
 
-# Import <package>._version_git.py without importing <package>
-path = glob.glob(__file__.replace("setup.py", "src/*/_version_git.py"))[0]
-spec = importlib.util.spec_from_file_location("_version_git", path)
-vg = importlib.util.module_from_spec(spec)
-spec.loader.exec_module(vg)
-
-setup(cmdclass=vg.get_cmdclass(), version=vg.__version__)
+setup(
+    use_scm_version={
+        "write_to": "src/scanspec/_version.py"
+    },
+    setup_requires=["setuptools>=45,<57", "setuptools_scm[toml]>=6.2", "wheel==0.33.1"],
```

Comment:
Can all of this go in setup.cfg instead?

Reply:
Actually, can it just be reduced to setuptools_scm[toml]>=6.2?

Reply:
In setup.cfg...

Reply:
I mean still leave it as you have it for pyproject.toml, but delete the wheel and setuptools references in setup.cfg?

Reply:
I am not sure that we need a pyproject.toml at all, but pipenv makes one if it doesn't exist with this content:
```
[build-system]
requires = ["setuptools", "wheel"]
build-backend = "setuptools.build_meta"
```

Reply:
Taking out the pinned wheel version has fixed [this](https://github.com/dls-controls/scanspec/runs/6214158036?check_suite_focus=true#step:3:105) error

Reply:
We have to have the setup_requires in setup.py or setup.cfg for Pipenv to work, so I suggest we have it in setup.cfg and have the minimal setup.py and pyproject.toml